Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
> But it is not expected that yarn would be able to find it before and would not be able to find it now.

If you're building from source & it's not working out the new location, then yes, that's an issue 😓.